wsgi/odyssy/announcement/api/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+
+from .views import (
+    AnnouncementListView,
+    AnnouncementParticularView,
+    AnnouncementTagView,
+    AnnouncementLatestView,
+)
+
+urlpatterns = [
+    url(r'^$', AnnouncementListView.as_view(),  name='all-announcement'),
```

Comment:
`name='all-announcement'` can we use some different name here as the name `name='all_announcement'` is already used in announcement app's urls and it might get confusing later.